Entities/ACrab.h

```diff
@@ -515,6 +515,18 @@ int FirearmActivationDelay() const;
 	void SetLimbPathPushForce(float force);
 
 
+	/// <summary>
+	/// Gets this ACrab's stride sound. Ownership is not transferred!
```

Comment:
Hey, sorry to do this to you, but the `not` should be `NOT`.
i.e. `/// Gets this ACrab's stride sound. Ownership is NOT transferred!`